index.html

```diff
@@ -81,13 +81,13 @@ <h3>LET’S CONNECT</h3>
         </article>
       </section>
     </div>
-
+    
     <section id="portfolio" title="portfolio-snapshot" class="card-works-box">
```

Comment:
- [ ] Kindly remove the contents of this section and generate the data dynamically from the array of objects you have in the javascript file as suggested by the previous reviewer. Please check the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) as it is stated there. 🤝️

**Project Requirement**
![Screenshot from 2022-12-21 19-15-29](https://user-images.githubusercontent.com/17243325/209073079-43bf62be-13d1-4475-bfcb-14d1662bf00e.png)

You can loop through the array of objects and generate the contents on page load.